pages/sdk/onramp/stripe.mdx

```diff
@@ -0,0 +1,203 @@
+import { Steps, Callout } from 'nextra/components'
+import Image from 'next/image'
+import StripeKycPersonalInfoImage from '../../../assets/stripe-kyc-personal-info.png'
+import KycAddress from '../../../assets/stripe-kyc-address.png'
+import PaymentMethod from '../../../assets/stripe-kyc-payment-method.png'
+
+# Onramp with Stripe
+
+<Callout type="info" emoji="ℹ️">
```

Comment:
Should this be removed as Germán has another PR open for this?

Reply:
I will remove it as we can always add it back.